src/components/Deleted.jsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+
+function RecentlyDeletedRooms() {
+  const deletedRooms = JSON.parse(localStorage.getItem('deletedRooms')) || [];
+
+  return (
+    <div className="deletedContainer">
+      <h1 id="deletedName">RECENTLY DELETED ROOMS</h1>
+      <ul id="ul">
+        {deletedRooms.map((room) => (
+          <li id="li" key={room.id}>
+            <h4 id="name">{room.name}</h4>
+            <div className="imageDiv">
+              <img id="imgOld" alt="room" src={room.image_url} />
+            </div>
+          </li>
+        ))}
+      </ul>
+    </div>
```

Comment:
![image](https://user-images.githubusercontent.com/61404403/234337215-17f7f5eb-0636-4648-8cb4-a8a3e42c18ec.png)
The page will look good when you include more detailed information about the rooms e.g `date reserved` etc